liquibase/liquibase.properties

```diff
@@ -0,0 +1,6 @@
+classpath=./mysql-connector-java-8.0.22.jar
+driver=com.mysql.cj.jdbc.Driver
+changeLogFile=dbchangelog.xml
+url=jdbc:mysql://localhost:3306/dinghy
```

Comment:
Will this need to change when we ship to on-prem/cloud?

Reply:
That file is for testing, at the end we will load values from the dinghy.yml and execute them with the command like `liquibase update --changeLogFile=dbchangelog.xml --user=user`